test/parallel/test-stream-writable-destroy.js

```diff
@@ -170,3 +170,19 @@ const { inherits } = require('util');
 
   new MyWritable();
 }
+
+{
+  // destroy and destroy callback
+  const write = new Writable({
+    write(chunk, enc, cb) { cb(); }
+  });
+
+  write.destroy();
+
+  const expected = new Error('kaboom')
```

Comment:
nope, I have some leftover files with linting errors under bench, and I didn't spot those :(. Updated